core/package.json

```diff
@@ -19,7 +19,6 @@
   "devDependencies": {
     "husky": "^8.0.1",
     "turbo": "^1.6.3",
-    "pnpm-deduplicate": "^0.4.1",
```

Comment:
@doubledup pnpm v7.26.0 supports built-in `pnpm dedupe` command so just remove here.